src/components/Quiz.js

```diff
@@ -0,0 +1,186 @@
+      Level: 1,
+      QuestionId: 2,
+      Options: [
+        { Text: "==", isCorrect: true },
+        { Text: "=", isCorrect: false },
+        { Text: "<>", isCorrect: false },
+        { Text: "><", isCorrect: false },
+      ],
+    },
+    {
+      Description: "Java: What langugage is this extension written in?",
+      Language: "java",
+      Level: 1,
+      QuestionId: 3,
+      Options: [
+        { Text: "Java", isCorrect: false },
+        { Text: "Python", isCorrect: false },
+        { Text: "SQL", isCorrect: false },
+        { Text: "JSX", isCorrect: true },
+      ],
+    },
+  ];
+
+  const [langugage, setLanguage] = useState("");
+  const [currentQuestion, setCurrentQuestion] = useState(0);
+  const [showScore, setShowScore] = useState(false);
+  const [score, setScore] = useState(0);
+  const [buttonColor, setButtonColor] = useState("white");
+  const [nextButtonDisplay, setNextButtonDisplay] = useState(true);
+
+  const handleAnswerOptionClick = (answerOption) => {
+    setButtonColor(
+      answerOption.isCorrect
+        ? "rgba(165, 214, 167, 1)"
+        : "rgba(239, 83, 80, 0.5)"
+    );
+    if (answerOption.isCorrect) {
+      setScore(score + 1);
+    }
+  };
+
+  const handleNextClick = () => {
+    const nextQuestion = currentQuestion + 1;
+    if (nextQuestion < questions.length) {
+      setCurrentQuestion(nextQuestion);
+    } else {
+      setNextButtonDisplay(nextButtonDisplay ? false : true);
+      setShowScore(true);
+    }
+  };
+
+  const QuizAgainButton = withRouter(({ history }) => (
+    <Button
+      primary
+      type="button"
+      onClick={() => {
+        history.push("/language");
+      }}
+    >
+      Quiz
+    </Button>
+  ));
+
+  const ExitButton = () => (
+    <button type="button" onClick={() => window.close()}>
+      Procrastinate
+    </button>
+  );
+
+  return (
+    <Container>
+      {showScore ? (
+        <div className="finish-display">
+          <p>Congratulations!</p>
+          <p>
+            You finished the quiz with {score}/{questions.length} correct
+            answers.
+          </p>
+          <div>
+            <ExitButton />
+            <QuizAgainButton />
+          </div>
+        </div>
+      ) : (
+        <>
+          <div className="question-section">
+            <div className="question-description">
+              {questions[currentQuestion].Description}
+            </div>
+          </div>
+          <div className="answer-section">
+            {questions[currentQuestion].Options.map((answerOption) => (
+              <button
+                style={{ backgroundColor: buttonColor }}
+                onClick={() => handleAnswerOptionClick(answerOption)}
+              >
+                {answerOption.Text}
+              </button>
+            ))}
+          </div>
+          <div>
+            <Button
+              primary
+              className="NextStep"
+              disabled={false}
+              onClick={() => handleNextClick()}
```

Comment:
When there's no parameters passed in, you can just use pass it in directly like this:
```suggestion
              onClick={handleNextClick}
```